mssql-plugin/pom.xml

```diff
@@ -100,20 +100,29 @@
       <plugin>
         <groupId>org.apache.felix</groupId>
         <artifactId>maven-bundle-plugin</artifactId>
+        <version>5.1.2</version>
+        <extensions>true</extensions>
         <configuration>
           <instructions>
             <_exportcontents>
-              io.cdap.plugin.*;
-              org.apache.commons.lang;
-              org.apache.commons.logging.*;
-              org.codehaus.jackson.*;
+              io.cdap.plugin.mssql.*;
+              io.cdap.plugin.db.batch.source.*;
+              io.cdap.plugin.db.batch.sink.*;
               com.microsoft.aad.*
```

Comment:
Do you know why need this is exported?

Reply:
Yet another unwanted export. Combing through all plugins to confirm all exports